packages/venia-concept/generateGraphQlFragmentTypes.js

```diff
@@ -0,0 +1,43 @@
+const fetch = require('node-fetch');
+const fs = require('fs');
+
+// Right now I've hardcoded this url because we only have process.env.MAGENTO_BACKEND_URL on run time
```

Comment:
Really? I thought when you run `yarn run build` it generates the `.env`. Can you not access that here?

Reply:
@sirugh nevermind managed to get the env config like so:
```
const config = require('dotenv').config();
const MAGENTO_BACKEND_URL = config.parsed.MAGENTO_BACKEND_URL;
```